.github/workflows/tox.yml

```diff
@@ -25,6 +26,9 @@ jobs:
         run: |
           set -euxo pipefail
           toxpyver=$(echo "${{ matrix.pyver }}" | tr -d .)
+          case "$toxpyver" in
+          *-alpha*|*-beta*) toxpyver=$(echo "$toxpyver" | sed 's/^.* \([1-9][0-9]*\)$/\1/') ;;
+          esac
```

Comment:
Just out of curiosity, what version string (pyver) is going to be supported by this addition? `something-beta version`?

Reply:
> Just out of curiosity, what version string (pyver) is going to be supported by this addition? `something-beta version`?

https://github.com/actions/setup-python - this describes the action

https://github.com/actions/python-versions/blob/main/versions-manifest.json - these are the python versions

The idea is that you can specify in the matrix a version like `"3.11.0-alpha - 3.11"` which means "any pre-release version in addition to any 3.11 released version".  But for some reason python 3.11 is not working in the github action vm (but it is working locally on f35).